utils/hyperlink.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Utility used to check if URL doesn't start with http:, http:, ? # or /, prepend http://
+ *
+ * @param {string} link Hyperlink to correct.
+ * @return {string} Hyperlink corrected.
+ */
+
```

Comment:
With the newline here, I don't know that JSDoc parsers would associate with the function. Suggest removal.